cfme/tests/infrastructure/test_host.py

```diff
@@ -87,6 +92,6 @@ def test_multiple_host_bad_creds(setup_provider, provider):
                            'validate_host': cfme_host["name"]})
 
     sel.click(credential_form.validate_multi_host)
-    flash.assert_message_match('Cannot complete login due to an incorrect user name or password.')
+    flash.assert_message_match(msgs[provider.type])
```

Comment:
@izapolsk we dont have a way to do a lookup by general provider category atm - so this use is valid and one-of is not usefull